utilities/src/biginteger/macros.rs

```diff
@@ -213,6 +213,13 @@ macro_rules! bigint_impl {
 
                 res
             }
+
+            fn from_slice(input: &[u64]) -> Self {
```

Comment:
I would prefer if we didn't add this method, as it is prone to abuse by Field impls, which require normalization to Montgomery form for any BigInteger

Reply:
Would it be OK to keep this, but only use it in field impls, enforcing a montgomery reduction? The problem this is trying to solve is instantiating a parameterized field from a slice.

Reply:
It also seems current `from_repr` implementation in fields forces a valid field.